src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  onSelect: (todo: Todo | null) => void;
+  selectedTodo: Todo | null;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onSelect }) => {
+  const [loading, setLoading] = useState(false);
```

Comment:
Since `loading` is being set in the beginning of each render, default value for the `loading` state should be `true`